### PR TITLE
Calculator addition

### PR DESCRIPTION
**Pull request by tomhebunn**
calculator logic defined within the store. Not hooked up to anything as yet but useful to have as reference for later

### CODE REVIEW

**Review by Exilliar — APPROVED** [review]
put in one small comment. But it's not blocking, so approving

**Review by psarras — COMMENTED** [review]
@tomhebunn if we are writing everything in `typescript` do we need the `get-pip.py` file?

**Comment by psarras** [comment]
Hey @tomhebunn , we have some conflicts after the other two PRs were merged! sorry!

File: src/store/utilities/carbonCalculator.ts [hunk 1]
```diff
@@ -0,0 +1,103 @@
+// DEFINITIONS
+// definitions to be superceded elsewhere once we hook up speckle
+interface speckleObject {
```
[the remark this answers is not in the document]
Comment (tomhebunn) [reply src/store/utilities/carbonCalculator.ts]: changed

**Review by psarras — APPROVED** [review]

**Pull request merged**